test/Battlefield.cpp

```diff
@@ -37,12 +36,12 @@ BOOST_AUTO_TEST_CASE(getNeighbouringTiles)
 	BOOST_TEST(neighbouringTiles.size()==6);
 
 	BOOST_REQUIRE(neighbouringTiles.size()==6 && mainHex==93);
-	BOOST_TEST(neighbouringTiles.at(0)==75);
-	BOOST_TEST(neighbouringTiles.at(1)==94);
-	BOOST_TEST(neighbouringTiles.at(2)==110);
-	BOOST_TEST(neighbouringTiles.at(3)==109);
-	BOOST_TEST(neighbouringTiles.at(4)==92);
-	BOOST_TEST(neighbouringTiles.at(5)==76);
+	BOOST_CHECK(std::find(neighbouringTiles.begin(), neighbouringTiles.end(), 75) != neighbouringTiles.end());
```

Comment:
std::find is too long, use vstd::contains. And order is important, so better to revert or check order in another test